app/views/developers/developers.scala.html

```diff
@@ -25,10 +25,6 @@
           width: 85%;
         }
 
-        .developer-status {
-          min-width: 150px;
-        }
-
```

Comment:
I think this is necessary to stop the column being collapsed to a very small size when a very VERY long email address is is displayed.